ops/ansible/secrets.yml

```diff
@@ -5,11 +5,38 @@
 
 deploy_discord_webhook_token: !vault |
```

Comment:
I think your choice to encrypt the entire URL (for `discord_webhook_url`) makes a lot more sense. We should do the same thing here instead of just encrypting the token.

Reply:
In the current discord module the parameters only support webhook_id and webhook_token.
 
https://docs.ansible.com/ansible/latest/collections/community/general/discord_module.html

discord_deploy_webhook_token can be used though instead of deploy_discord_webhook_token to converge on naming pattern